### PR TITLE
Ticket 4 - View a list of all articles

### PR DESCRIPTION
**Pull request by 1991hannah**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const ArticleCard = (({article}) => {
+    return(
+        <section className="article-card">
+            <div className="article-image">
```
Comment (RichardNeat) [inline src/components/ArticleCard.jsx]: Try and avoid using divs if you can :)

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Navbar from "./Navbar"
+
+const Header = (() => {
+    return(
+        <header className="header-container">
+        <h1 className="header">This is the Header</h1>
+        <Navbar />
```
Comment (RichardNeat) [inline src/components/Header.jsx]: Does the Navbar component need to be a child of the Header component? What was the reason for this?

**Review by RichardNeat — APPROVED** [review]
Really nice Hannah! Just a couple of comments below to consider please, once you're happy you can go ahead and merge :)

File: src/components/ArticleCard.jsx (hunk 1 above)
Comment (1991hannah) [reply src/components/ArticleCard.jsx]: Thanks Rich! Have taken the divs out 👍🏻

File: src/components/Header.jsx (hunk 2 above)
Comment (1991hannah) [reply src/components/Header.jsx]: Probably not actually!

**Pull request merged**